### DISCUSSION TITLE
Help me understand the following PROMPT parameters ${ret_status} and %p

### DISCUSSION THREAD

**Discussion by paultheman** [ohmyzsh/ohmyzsh#11646] [Help] [answered]
Although I searched a lot I could not find an answer.
`PROMPT='${ret_status}%{$fg_bold[green]%}%p %{$fg[cyan]%}%c %{$fg_bold[blue]%}$(git_prompt_info)%{$fg_bold[blue]%} %D %T % %{$reset_color%}'
`
Thank you!

**Comment by paultheman** [comment, marked as answer]
${ret_status} was previously a local variable in robbierussel theme. It is now removed.

**Reply by mcornella** [reply]
Also `%p` does nothing, can be safely removed. See [valid prompt sequences](https://zsh.sourceforge.io/Doc/Release/Prompt-Expansion.html).
